translations/en-us.json

```
@@ -53,12 +53,14 @@
   "header_background_color_description": "Color of the background color for the header",
   "ocw_label": "OCW URL",
   "ocw_description": "Specify the URL for the OCW Page",
-  "ocw_creative_common_label": "OCW Creative Common URL",
-  "ocw_creative_common_description": "Speacify OCW Creative Common page url",
+  "ocw_creative_common_label": "OCW Creative Commons URL",
+  "ocw_creative_common_description": "Specify OCW Creative Commons page url",
   "ocw_accessibility_label": "OCW Accessibility Page",
-  "ocw_accessibility_description": "Speacify OCW Accessibility page url",
-  "contact_us_label": "Contact Us URL ",
+  "ocw_accessibility_description": "Specify OCW Accessibility page url",
+  "contact_us_label": "Contact Us URL",
   "contact_us_description": "Specify Contact Us Page URL",
+  "gtm_id_label": "Specify the Google Tag Manager ID",
+  "gtm_id_description": "Google Tag Manager ID",
```

I think the correct order would be:

```suggestion
  "gtm_id_label": "Google Tag Manager ID",
  "gtm_id_description": "Specify the Google Tag Manager ID",
```